gnomad/utils/vcf.py

```diff
@@ -180,6 +189,18 @@
             " variants for VQSR"
         )
     },
+    "positive_train_site": {
```

Comment:
do POSITIVE_TRAIN_SITE and NEGATIVE_TRAIN_SITE above need to be deleted?

Reply:
I left it as to not break any pipeline that looks in the dictionary for it, like v3